src/docs/reference/backups.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Backups
+---
+
+Backups are a feature that enables point in time data recovery for [volumes](/reference/volumes) on Railway.
+
+## How it works
+
+When mounted to a service, backups can be manually created, deleted and restored. And they can also work on a Daily/Weekly/Monthly schedule.
+
+Schedules are beta and exist behind a feature-flag attached to the service.
+
+## Retention
+
+- Daily backups are taken every day and are kept for 6 days
+- Weekly backups are taken every week and are kept for 4 weeks
+- Monthly backups are taken every month and are kept for 3 months
+
+## Pricing
+
+Backups are incremental and Copy-on-Write, we only charge for the data exclusive to them, that aren't from other snapshots or the volume itself.
+
+Backups are billed like volumes, at **$0.25 / GB**, billed monthly.
+
+## Caveats
```

Comment:
Another one, you can't restore a backup to a service in another environment.

Reply:
Done!